COMMPLAN.md

```diff
@@ -0,0 +1,207 @@
+highlighting, which makes it perfect for instant messaging about
+software issues, general software use questions, or other technical
+discussions.
+
+Zulip has multiple *Streams* which are designed to topically target
+certain messages.  You can create *Topics* within each of these
+Streams, much like subject headings in email.
+
+#### What should we use it for?
+
+Use Zulip for NCAR-wide communication.  Note that Zulip is accessible
+by *invitation only*, which means it really only includes NCAR staff
+and close collaborators.
+
+Use Zulip to post and answer public questions about our services
+(e.g., the NCAR JupyterHub) or software (e.g., "How do I do X with Y?").
+Some people on Zulip may feel uncomfortable asking questions on other
+messaging/chat platforms that other software communities use, such as
+Gitter or StackOverflow, and they may feel more comfortable asking
+their questions on Zulip first.  Some of their questions may relate
+more to the environment that we provide for them to *use* the software,
+which is another reason why they may ask their questions on Zulip first.
+In general, Zulip is the first line of question/answer for staff here
+at NCAR on Python topics.
+
+Public Streams (i.e., those labeled with a `#` symbol) can be "joined"
+by *anyone* on Zulip.  Hence, they should be used to ask, or answer,
+questions, or make comments of public interest, related to the Public
+Stream subject.  For example, the `dask` Stream should be used for any
+Dask-specific issues or questions. The `jupyter` Stream should be used
+for any Jupyter (Lab or Hub) issues or questions.  The `python-dev` and
+`python-questions` Public Streams are *catch-all* streams for any
+Python-related discussion that doesn't fall into the more specific
+public streams (e.g., `jupyter`, `dask`).  NCAR-wide *announcements*
+should be made to the `announce` Public Stream.  In general, the name
+of the Public Stream defines the *subject* of its discussions.
+
+Private Streams (i.e., those labeled with a lock symbol) should be used
+for *internal* communications between people who have been invited to
+the Private Stream.  For example, the `Xdev` Private Stream is meant
+for internal communication within the Xdev Team.  In general, the
+name of the Private Stream defines the *group of people* included in
+the discussions.
+
+Zulip is also excellent for instant messaging *in general*, so if you
+are going to be watching Zulip to help out other Python users here at
+NCAR, then it makes sense to use it for direct communication amongst
+Xdev Team members, too.
+
+#### What should we *not* use it for?
+
+Don't use Zulip for communication that is meant for the
+*entire user community* (e.g., all of Pangeo), and not just NCAR staff.
+In those cases, I recommend using the Pangeo communication channels, as
+described in the [PANGEO Guide](PANGEO.md).
+
+### The Xdev Blog
+
+#### Why should we use it?
+
+Our blog (https://ncar.github.io/xdev) is an excellent way for us to
```

Comment:
One last thing caught my eye -- how about starting this section with "[Our blog](https://ncar.github.io/xdev) is an..." instead of including the URL in the parenthetical?